_Boards/Atmel/Board_ProMicro/MFBoards.h

```diff
@@ -17,14 +17,15 @@
 #define MF_STEPPER_SUPPORT   1
 #endif
 #ifndef MF_SERVO_SUPPORT
-#define MF_SERVO_SUPPORT 1
+#define MF_SERVO_SUPPORT     0
```

Comment:
Was it intentional to remove servo support from the pro micro?

Reply:
No, this change (and other ones below) have been introduced by inadvertently basing on a local main branch that had strayed from the  official main. Sorry for the mishap, fixed now.